dev-python/dbuspython/dbuspython-1.2.14.recipe

```diff
@@ -0,0 +1,110 @@
+SUMMARY="The original dbus implementation for Python"
+DESCRIPTION="dbus-python is the original Python binding \
+for dbus, the reference implementation of the D-Bus protocol."
+HOMEPAGE="https://www.freedesktop.org/wiki/Software/DBusBindings/#python"
+COPYRIGHT="\
+		2002-2010 Red Hat Inc.
+		2003 David Zeuthen
+		2002-2003 CodeFactory AB
+		2003 James Willcox
+		2004 Rob Taylor
+		2004 Anders Carlsson
+		2005 Colin Walters
+		2006 Steve Frécinaux
+		2009-2011 Nokia Corporation
+		2010-2012 Mike Gorse
+		2016 Simon McVittie
+		2005-2019 Collabora Ltd."
+LICENSE="MIT
+		AFL v2.1
+		GNU GPL v2"
```

Comment:
We use one tab, please format it like in this recipe: https://github.com/haikuports/haikuports/blob/2a1524520de2e197f1bee57cb031920ec2721467/app-text/mandoc/mandoc-1.14.3.recipe#L11-L29